manifests/plugin/monitoring/icinga2.pp

```diff
@@ -0,0 +1,49 @@
+# = Foreman Proxy Icinga 2 Monitoring plugin
+#
+# This class installs the Icinga 2 Monitoring plugin for Foreman proxy
+#
+# === Parameters:
+#
+# $enabled::               Enable this plugin.
```

Comment:
Move `enabled` into an Advanced section like with the main monitoring class, so it's hidden by default in the installer.